### ISSUE TITLE
write_packages should write the packages.R file in alphabetical order

### ISSUE THREAD

**Issue by njtierney** [njtierney/codelens#11]

**Comment by dhduncan** [comment]
Repeat or move the `sort` (and `unique`) operations further down to after you glue conflicted into the object?  

As an aside (are asides allowed in issues?), the above would largely make redundant an issue I was about to add about when someone (foolishly) runs this operation on a directory that already contains a "packages.R" file.

**Comment by dhduncan** [comment]
```unique_libs <- sort(unique(unlist(deps$library)))
  unique_libs
  augmented_libs <- c(glue::glue("library({unique_libs})"), 
    "library(conflicted)")
libs_to_write <- sort(unique(augmented_libs))
```

**Comment by njtierney** [comment]
> the above would largely make redundant an issue I was about to add about when someone (foolishly) runs this operation on a directory that already contains a "packages.R" file.

Yes this should definitely make some noise when `packages.R` already exists, but I would write that as a separate issue.
